tests/test_main.py

```
@@ -0,0 +1,49 @@
+
+        # Arrange
+        mock_time.side_effect = [0, 0.05, 0.1]  # Time values for FPS calculation
+        
+        # Act
+        main.main(MagicMock())
+        
+        # Assert
+        self.assertTrue(True)  # If we got here, no exceptions were raised
+
+    @patch('curses.has_colors')
+    def test_init_curses(self, mock_has_colors):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_init_curses`

Add return type annotation: `None`

(ANN201)